apps/angular/src/core/utils/EmptyValuePipe.ts

```diff
@@ -0,0 +1,15 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+/** Pipe to display 'Unknown' in case of an empty string. */
+@Pipe({
+	name: 'emptyValue',
+})
+export class EmptyValue implements PipeTransform {
+	/**
+	 * @param value Original string.
+	 * @returns Original string or 'unknown.
+	 */
+	public transform(value: string): string {
+		return value || 'Unknown';
```

Comment:
Would be great to be more explicit in your conditions
```suggestion
		return value === '' || 'Unknown';
```